core/templates/domain/topic/editable-topic-backend-api.service.ts

```diff
@@ -47,6 +47,8 @@ angular.module('oppia').factory('EditableTopicBackendApiService', [
               response.data.grouped_skill_summary_dicts),
             skillIdToDescriptionDict: angular.copy(
               response.data.skill_id_to_description_dict),
+            skillQuestionCountDict: angular.copy(
```

Comment:
Let's try to minimize the use of `angular.copy(..)` -- it has some performance issues ([#11099](https://github.com/angular/angular.js/issues/11099)). In this case, the value seems to be a simple object, so we could use the [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to copy over the properties to a new object. WDYT?

eg.
```suggestion
            skillQuestionCountDict: { ...response.data.skill_question_count_dict }
```

Reply:
Oh I didnt knew it was inefficent, Fixed.